plugins/outputs/http/http.go.orig

```diff
@@ -0,0 +1,371 @@
+package http
```

Comment:
remove?

Reply:
I rewrote history to avoid the one separate file removal commit 🤷🏼‍♂️